app/kubemci/cmd/create.go

```diff
@@ -87,6 +87,12 @@ type CreateOptions struct {
 	// the resource does not exist, or is already the correct
 	// value, then 'force' is a no-op.
 	ForceUpdate bool
+	// Name of the namespace for the ingress. Overrides the namespace in the IngressFilename (if present).
+	// Optional.
+	namespace string
+	// Static IP name for the ingress. Overrides the annotation in the IngressFilename (if present).
+	// Optional.
+	StaticIPName string
```

Comment:
This can be private as well (same as namespace)?

Reply:
Will leave global privatization to you for future work.